### ISSUE TITLE
RwGsClassVersioningSymbolDictPatch>>updatePatchesForNewClassVersion: can trigger an error...

### ISSUE THREAD

**Issue by dalehenrich** [dalehenrich/Rowan#114]
when there is more than one package involved in the load ... should wire out that test

**Comment by dalehenrich** [comment]
The error message in `RwGsClassVersioningSymbolDictPatch >> updatePatchesForNewClassVersion:` comes about when there is a new class version in one package with additional changes in other packages that aren't related.

**Comment by dalehenrich** [comment]
Okay, it turns that the method classesModified: implemented in the following classes:
```
RwEntitySetModification>>classesModifiedDo:
 RwPackageSetModification>>classesModifiedDo:
 RwProjectSetModification>>classesModifiedDo:
```
does not include a packageDefintion as an argument to the block (nor does it include a projectDefinition) ...  this bug (and probably others) is caused by the fact that the methods in subclasses of classes with new versions will end up being put into the wrong package ... if you are unlucky, the system won't trip across this discrepancy and if you are lucky you'll get an error when the method is added twice (RwRowanProjectIssuesTest>>testIssue114_bothModificationsMustShareCommonAfter_3)

**Comment by dalehenrich** [comment]
merged to master (#112)
